doc/release/1.6.3-notes.rst

```diff
@@ -10,11 +10,22 @@ compared to 1.6.2.
 Authors
 =======
 
+* Peter Bell
+* Tyler Reddy
+* 刘星雨 +
```

Comment:
@charlotte12l  Did our script do the right thing with your name here?

If not, I can adjust this.

Reply:
That's my name in Chinese, you can modify it to the English version: Xingyu Liu. Thanks!